src/x86/intel_joule_expansion.c

```diff
@@ -367,12 +368,10 @@ mraa_joule_expansion_board()
     b->pins[pos].pwm.mux_total = 0;
     pos++;
 
-    // pin 31
-    strncpy(b->pins[pos].name, "ISH_IO2", 8);
-    b->pins[pos].capabilities = (mraa_pincapabilities_t){ 1, 1, 0, 0, 0, 0, 0, 0 };
-    // High level will be V1P8 - VBE on MBT3904D
-    b->pins[pos].gpio.pinmap = 339;
-    b->pins[pos].gpio.mux_total = 0;
+    strncpy(b->pins[pos].name, "I2S3SDO", 8);
```

Comment:
Please revert this, or at least leave the GPIO function and OS map as is with the new pin name.